### ISSUE TITLE
Remove dirty tracking pixels

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#3827]
I visited our app without ublock & saw that we're loading facebook & linkedin tracking pixels.
letting zuck spy on our users gives me a great deal of tension.

Are we getting any value from allowing this? If not, we need to remove. If so, we need to discuss, and then probably remove.

AC
- [ ] no tracking pixels loaded

Estimate: 8 hours

**Comment by ackernaut** [comment]
hear, hear!!

**Comment by avivapinchas** [comment]
> Are we getting any value from allowing this? 

We've just started testing ads on Facebook and LinkedIn. Tracking conversions on our site allows us to optimize our spend, so that we're paying for traffic that's actually converting. The value we get from tracking is an efficient, effective ad spend. 

Advertising in general allows us to test messaging, learn about our users and increase our users. 
 
If we chose not to advertise on Facebook, we'll be giving up a non-trivial amount of users - it's one of the top two ad platforms. 

> letting zuck spy on our users gives me a great deal of tension.
Seeing an issue about our ad tracking logged as a bug without me being tagged or consulted gives me a great deal of tension too. 

Also worth noting: Microsoft (not FB) owns LinkedIn, and we're already using GitHub.

Is the tension here FB-specific, a general distaste for ads, or something else?
